kratos/utilities/specifications_utilities.h

```diff
@@ -128,6 +134,28 @@ class KRATOS_API(KRATOS_CORE) SpecificationsUtilities
         ALE        = 2
     };
 
+    ///@}
+    ///@name Life Cycle
+    ///@{
+
+    /**
+     * @brief The default constructor
+     */
+    SpecificationsUtilities() = default;
```

Comment:
```suggestion
    SpecificationsUtilities() = delete;
```

this should not be instantiable